pkg/catalogsourceconfig/configuring.go

```diff
@@ -21,11 +20,16 @@ import (
 // TBD: Vendor in the folder once we require more than just these constants from
 // the OLM registry code.
```

Comment:
nit: This comment seems to be disingenuous since we are adding to the const list. We don't need to vendor, but maybe the comment should be updated?

Reply:
Actually the constants need to be removed :-)